### DISCUSSION TITLE
Tab coordinates in the cell geometry - Point of reference?

### DISCUSSION THREAD

**Discussion by abhishek-appana** [pybamm-team/PyBaMM#4094] [Q&A]
Hello everyone! I have a few questions regarding the tab positions of a pouch cell!

- Can someone help me understand where the point of reference (origin) is when I need to specify the centre y and z coordinates of the positive and negative tabs? I could not find this in any documentation
- Do these coordinates affect the heat generation/transfer in the thermal submodels? I did not see any equation in thermal submodels which uses this data.
- My battery has a counter tab configuration and I am using the 2D current collector model. Does this submodel still work for this configuration?

Thank you for the advice!

**Comment by abhishek-appana** [comment]
Hello everyone!

Can someone help me with this?

Thank you!

**Reply by JDS19** [reply]
Hi, did you ever end up figuring out how to interpret these parameters? I'm wondering the same thing. The standard from Marquis2019 shows an electrode height of 0.137 m and width of 0.207 m; the y- and z- center coordinates are 0.147 m and 0.137 m, respectively. My guess is then that the origin is in the bottom left corner when looking at the side of the current collector not in contact with the electrode.
